### ISSUE TITLE
Deployment job is failing

### ISSUE THREAD

**Issue by tsalo** [PennLINC/qsirecon#50]
## Summary

The `build_and_deploy` step of the CircleCI workflow is failing.
The problem seems to come from the QSIRecon Docker wrapper:
```
sed: can't read wrapper/qsirecon_container/qsirecon_docker.py: No such file or directory
```

**Comment by tsalo** [comment]
Since I merged #52, the deployment job is passing, but it's not actually deploying to DockerHub. Looks like CircleCI was missing the Docker-related environment variables. I've added my credentials and resubmitted the job, so hopefully it deploys this time.

**Comment by tsalo** [comment]
We're up! https://hub.docker.com/r/pennlinc/qsirecon/tags/
